agent/action/pipeline_manager.py

```
@@ -1,807 +1,464 @@
+                if not targets:
+                    utils.mfaalog.info("[Py] 🧹 本次任务没有需要还原的节点")
+                    _side_click(context, argv, params.get("click"))
+                    return True
+            else:
+                targets = _targets(context, spec, allow_regex=False)
+
+            done, missing = [], []
+            for node in targets:
+                payload = ledger.pop(node, None)
+                if payload is None:
+                    # 没被改过 / 已还原过 / 是新建的节点 —— 还原是幂等的，不算失败
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)